release-notes/2022-12/README.md

```diff
@@ -47,6 +47,8 @@ The 2022-12 release bundle for SystemLink Enterprise has been published to <http
 - `notification 0.1.0`
     - Newly included chart
     - You must configure a secret for MongoDB credentials in `notification.secrets.mongodb`
+- `sl-jupyterhub 0.2.0`
```

Comment:
Should this value be added to the [getting started template](https://github.com/ni/install-systemlink-enterprise/blob/main/getting-started/templates/systemlink-values.yaml) with a comment and perhaps an ATTENTION tag so that the user's attention is called to this choice.

Reply:
updated